taglib-core/src/main/java/com/reevoo/client/ReevooMarkClient.java

```diff
@@ -20,9 +21,21 @@
  */
 public class ReevooMarkClient {
     protected HttpClient client;
-
-    public ReevooMarkClient(int connectTimeout, String proxyHost, String proxyPort) {
-        this.client = HttpClientFactory.build(connectTimeout, proxyHost, proxyPort);
+    private Properties config;
+    private int failedRequestCacheTime;
+    private int circuitBreakerWaitTime;
+    private int circuitBreakerLimit;
+
+    public ReevooMarkClient(Properties config) {
+        this.config = config;
+        this.client = HttpClientFactory.build(
+                Integer.valueOf(config.getProperty("http.timeout")),
+                config.getProperty("http.proxyHost"),
+                config.getProperty("http.proxyPort"));
```

Comment:
i would close the bracket on the next line so you can see the arguments passed more clear